framework/db/oci/QueryBuilder.php

```
@@ -229,90 +251,202 @@ public function addForeignKey($name, $table, $columns, $refTable, $refColumns, $
             $constraintCondition = ['and'];
+
             foreach ($constraint->columnNames as $name) {
                 $quotedName = $this->db->quoteColumnName($name);
+
                 $constraintCondition[] = "$quotedTableName.$quotedName=\"EXCLUDED\".$quotedName";
             }
+
             $onCondition[] = $constraintCondition;
         }
+
         $on = $this->buildCondition($onCondition, $params);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $on. Configured minimum length is 3. (undefined)

(ShortVariable)